### PR TITLE
Adding Basic Implementation Of Graph

### PR DESCRIPTION
**Pull request by shivamk23**
### Description
This is a basic representation of Graphs , and you can extend it Based on your requirements. You might want to create separate classes for vertices and edges, implement a direct graph, or add additional functionality such as depth-first search or breadth-first-search.

### Related/Fixes Issue
 Issue : #67

### Changes Made
I have added a basic implementation of graph which any beginner can easily able to understand.

1. The add Edge method is used to add an undirected edge between two vertices.
2. The print Graph method uses a lambda expression for concise printing.
3. The Main class creates an instance of the Graph class and adds edges to it.

### CODE REVIEW

**Comment by rohan472000** [comment]
Add description and Wikipedia links under comments in.java files

**Comment by shivamk23** [comment]
okay sure @rohan472000

**Comment by shivamk23** [comment]
Done @rohan472000, now please merge my commit.

**Review by rohan472000 — CHANGES_REQUESTED** [review]
Keep those description and link at top now in middle.

**Comment by shivamk23** [comment]
Added description and link at top. @rohan472000

**Comment by rohan472000** [comment]
You have proposed two .java files of graph...whats the difference in them?

**Comment by rohan472000** [comment]
> Sorry @rohan472000 it's my mistake, can you please merge my code. if everything is alright.

why so desperate to get your code merged?? after review and correction it will be merged.

File: Graphs/ImplementingGraphs.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+   Relevant Links:
+   - Graph Theory: https://en.wikipedia.org/wiki/Graph_theory
+   - Graph (Discrete Mathematics): https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)
+   - Adjacency List: https://en.wikipedia.org/wiki/Adjacency_list
+
+   Usage:
+   1. Create a Graph object with the desired number of vertices.
+   2. Add edges using the addEdge method.
+   3. Print the graph's adjacency list using the printGraph method.
```
Comment (rohan472000) [inline Graphs/ImplementingGraphs.java:20]:
```suggestion
```

**Review by rohan472000 — APPROVED** [review]

**Pull request merged**